src/decaylanguage/modeling/amplitudechain.py

```diff
@@ -55,8 +56,8 @@ class AmplitudeChain(ModelDecay):
     name = attr.ib(None)
 
     # Class members keep track of additions
-    all_particles = set()
-    final_particles = set()
+    all_particles: ClassVar[Particle] = set()
+    final_particles: ClassVar[Particle] = set()
```

Comment:
Technically, I think this should be `ClassVar[set[Particle]]`. Though just plain `ClassVar` might be inferred correctly (probably not, I've not had much success with `Final`'s arguments being inferred by all type checkers, notably pyright).